packages/ui/src/components/Tooltip.stories.tsx

```diff
@@ -0,0 +1,179 @@
+import { select, withKnobs } from "@storybook/addon-knobs";
+
+import React, { useRef, useState } from "react";
+import { Pressable, View } from "react-native";
+import Hoverable from "./Hoverable";
+import Tooltip from "./Tooltip";
+
+export default {
+  title: "Tooltip",
+  component: Tooltip,
+  decorators: [withKnobs],
+};
+
+export function Text() {
+  const [isHovered, setHovered] = useState(false);
+  const ref = useRef<View | null>(null);
+  return (
+    <Hoverable
+      onHoverIn={() => setHovered(true)}
+      onHoverOut={() => setHovered(false)}
+    >
+      <View
+        ref={ref}
+        style={{
+          borderWidth: 1,
```

Comment:
Just FYI: the `border` shorthand property doesn't exist in React Native (adds unnecessary parsing overhead… and breaks the type checker! :) )